examples/manifests/interpolation.yml

```diff
@@ -0,0 +1,26 @@
+name: interpolation-demo
+description: simple demo of interpolation plugin
+tags:
+initialize:
+  plugins:
+    interpolation:
+      method: Interpolation
+      path: "builtin"
+      global-config:
+        method: linear
+        x: [0, 10, 50, 100]
+        y: [0.12, 0.32, 0.75, 1.02]
+        input-parameter: "cpu/utilization"
+        output-parameter: "result"
+
+tree:
+  children:
+    child:
+      pipeline:
+        - interpolation
+      inputs:
+        - timestamp: 2023-07-06T00:00
+          duration: 3600
+          cpu/utilization: 45
+          # vcpus-allocated: 4
```

Comment:
should they stay commented?

Reply:
thanks for noticing, the manifest moved to the manifests/plugins folder